### PR TITLE
Docs: added carbon ad

### PR DESCRIPTION
**Pull request by anikethsaha**
Added CarbonAds

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -19,6 +19,108 @@
...
+			}
+
+			.carbon-img img {
+				display: block;
+				margin: 0 auto;
+				max-width: 130px;
+				width: 130px;
+				height: auto;
+			}
+		}
```
Comment (sy-records) [inline docs/index.html:121]: Can we create it as a css file? for use in both locations.

Comment (anikethsaha) [reply docs/index.html:121]: I tried, for some reason it's not loading that file.

Comment (sy-records) [reply docs/index.html:121]: well, I try it.

Comment (anikethsaha) [reply docs/index.html:121]: there are few [here](https://codepen.io/collection/AyEoKk/?__cf_chl_jschl_tk__=90a2969943902982e6f4af8d84f0c9ffc2831e7b-1597560043-0-AfrHJDIz9I3eXlHh5jILZCelQdj_BRK98h8PDoXhnfW9IoNyglegU9XdfIliC_pFReugMXn7O3A85VL258ETHodRBcms_ZpHf1-mEBzvDco3oSopfcnUFV4tohgVkpF3GjU3zAecDp_VIUElfnCTyBrWYysT0ZePyiFj_jK1C9uzjVKiGKBgTCVE6X83_66hUNKFQ7F8kJdUeS8NbrFU0nSC2A8kCeE7GlZit7BYIKESXzVoSsePiACdlhOzzqTrkQ8jywsbZOEFOS93VoBJF2ZbS3nu3u4MBm6fzkyQNviIW3bXmRwadEchtgwP8E4g_N7dhsdAuj_30-1KLFyo0TB2HhAMd6MZJOciSRqr00PY&cursor=ZD0wJm89MSZwPTEmdj00NzE4MjY5) for reference.

Comment (sy-records) [reply docs/index.html:121]: > I tried, for some reason it's not loading that file.

I tried it, and it's true...

**Comment by anikethsaha** [comment]
agreed let's use the plugin

**Comment by anikethsaha** [comment]
We will need to release a version once it is merged in `develop`, I can directly submit a PR against `master` as there would be conflicts.

**Comment by anikethsaha** [comment]
> I m just working on it also, it seems a litte big.

yeah if PR against `master` then the diff is quite big, We need to release it at the earliest

**Pull request merged**